pyvista/plotting/itkplotter.py

```diff
@@ -1,8 +1,7 @@
-"""PyVista-like ITKwidgets plotter.
-
-This is super experimental: use with caution.
-"""
+"""PyVista-like ITKwidgets plotter."""
+from packaging import version
```

Comment:
I'm getting:
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-c4d0832af397> in <module>
----> 1 import pyvista as pv
      2 
      3 pv.plot_itk(pv.Sphere())

~/Software/pyvista/pyvista/pyvista/__init__.py in <module>
      3 import warnings
      4 from pyvista._version import __version__
----> 5 from pyvista.plotting import *
      6 from pyvista.utilities import *
      7 from pyvista.core import *

~/Software/pyvista/pyvista/pyvista/plotting/__init__.py in <module>
     14 from .qt_plotting import BackgroundPlotter, QtInteractor, MainWindow, Counter
     15 from .helpers import plot, plot_arrows, plot_compare_four, plot_itk
---> 16 from .itkplotter import PlotterITK

~/Software/pyvista/pyvista/pyvista/plotting/itkplotter.py in <module>
      1 """PyVista-like ITKwidgets plotter."""
----> 2 from packaging import version
      3 
      4 import numpy as np
      5 import pyvista as pv

ModuleNotFoundError: No module named 'packaging'
```

if this is not a standard package, then we should maybe avoid using it